config/routes.rb

```diff
@@ -1,13 +1,14 @@
-require "sidekiq/web"
-Rails.application.routes.draw do
+# frozen_string_literal: true
+
+require 'sidekiq/web'
+require 'sidekiq-scheduler/web'
 
-  authenticate :user, lambda { |u| u.roles.include?("admin") } do
+Rails.application.routes.draw do
+  authenticate :admin do
     mount Sidekiq::Web => '/sidekiq'
   end
 
-  if Rails.env.development?
-    mount LetterOpenerWeb::Engine, at: "/letter_opener"
-  end
+  mount LetterOpenerWeb::Engine, at: '/letter_opener' # if Rails.env.development?
```

Comment:
Hi @MoretS, 

Is it necessary to keep the letter_opener route for production environment ?

```suggestion
  mount LetterOpenerWeb::Engine, at: '/letter_opener' if Rails.env.development?
```